programs/clearing_house/src/state/insurance_fund_stake.rs

```diff
@@ -0,0 +1,17 @@
+use anchor_lang::prelude::*;
+
+#[account(zero_copy)]
+#[derive(Default, Eq, PartialEq, Debug)]
+#[repr(packed)]
+pub struct InsuranceFundStake {
+    pub authority: Pubkey,
+    pub bank_index: u64,
+
+    pub lp_shares: u128,
+
+    pub last_withdraw_request_shares: u128, // get zero as 0 when not in escrow
+    pub last_withdraw_request_value: u64,
+    pub last_withdraw_request_ts: i64,
+
+    pub cost_basis: i128,
```

Comment:
didnt immediately understand why you need this, can you explain?

Reply:
its not used anywhere, was just tracking